src/app/layout/footer/footer.component.html

```diff
@@ -0,0 +1,5 @@
+<footer class="footer bg-light text-center text-lg-start">
+  <div class="text-center p-3" style="background-color: rgba(0, 0, 0, 0.2);">
```

Comment:
Implementing `inline css` is not a good idea. Please try to follow `BEM strategies` and write reusable styles as much as possible.